pkg/kubectl/history.go

```diff
@@ -36,8 +36,8 @@ import (
 	clientappsv1 "k8s.io/client-go/kubernetes/typed/apps/v1"
 	api "k8s.io/kubernetes/pkg/apis/core"
 	apiv1 "k8s.io/kubernetes/pkg/apis/core/v1"
-	deploymentutil "k8s.io/kubernetes/pkg/controller/deployment/util"
 	kapps "k8s.io/kubernetes/pkg/kubectl/apps"
+	"k8s.io/kubernetes/pkg/kubectl/util/deploymentutil"
```

Comment:
I know we've landed `podutils`, but it bothers me, how about we call this `pkg/kubectl/util/deployment` it'll be shorter and straightforward and doesn't repeat that nasty util word.

Reply:
I've changed the directory/package to "util/deployment". But I had to use an import alias of "deploymentutil" because there were collisions with variables named "deployment". Please let me know what you think.

Reply:
> I've changed the directory/package to "util/deployment". But I had to use an import alias of "deploymentutil" because there were collisions with variables named "deployment"

I think that's fine :+1: